core/sys/types/src/lib.rs

```diff
@@ -10,4 +10,21 @@
     non_upper_case_globals
 )]
 
+use core::fmt;
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+
+impl fmt::Debug for sgx_attributes_t {
```

Comment:
I'm not fond of this, I wanted to have the tests to show the wrapping worked.
I want to look into forcing bindgen to generate debug and Eq for structs, not sure why it's not, it seems to only be doing it for simple primitive aliases

Reply:
this is taken care of in #91